test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -688,48 +688,103 @@
 #      initPods: 20000
 #      measurePods: 5000
 
-- name: Unschedulable
+# Measure the time it takes to schedule default measurePods after scheduling 20% of high priority Pods that trigger preemption.
+# The measurePods are expected to be scheduled after scheduler handled the higher priority Pods. It's expected that running the apiserver updates asynchronously https://github.com/kubernetes/kubernetes/issues/126858 (once fixed), should significantly speed up this test.
+- name: PreemptionBlocking
   workloadTemplate:
   - opcode: createNodes
     countParam: $initNodes
   - opcode: createPods
     countParam: $initPods
-    podTemplatePath: config/templates/pod-large-cpu.yaml
+    podTemplatePath: config/templates/pod-low-priority.yaml
+    # Collect metrics from all createPods ops.
+  - opcode: startCollectingMetrics
+    name: preemptionBlockingMetric
+    namespaces: [blocking, measure]
+  - opcode: createPods
+    countParam: $blockingPods
+    podTemplatePath: config/templates/pod-high-priority.yaml
+    skipWaitToCompletion: true
+    namespace: blocking
+  - opcode: createPods
+    countParam: $measurePods
+    namespace: measure
+  - opcode: stopCollectingMetrics
+  workloads:
+  - name: 5Nodes
+    labels: [integration-test, performance, short]
+    params:
+      initNodes: 5
+      initPods: 20
+      blockingPods: 1
+      measurePods: 4
+  - name: 500Nodes
+    labels: [performance, short]
+    threshold: 18  # TODO Adjust the threshold
+    params:
+      initNodes: 500
+      initPods: 2000
+      blockingPods: 100
+      measurePods: 400
+  - name: 5000Nodes
+    labels: [performance]
+    params:
+      initNodes: 5000
+      initPods: 20000
+      blockingPods: 1000
+      measurePods: 4000
+
+
+# Measure the time it takes to schedule default measurePods after scheduling 20% of high priority Pods that becomes unschedulable due to the resource constraints.
+# The measurePods are expected to be scheduled after scheduler handled the higher priority Pods. It's expected that running the apiserver updates asynchronously https://github.com/kubernetes/kubernetes/issues/126858 (once fixed), should significantly speed up this test.
+- name: UnschedulableBlocking
+  workloadTemplate:
+  - opcode: createNodes
+    countParam: $initNodes
+  # Collect metrics from all createPods ops.
+  - opcode: startCollectingMetrics
+    name: preemptionBlockingMetric
+    namespaces: [blocking, measure]
```

Comment:
In general, each operation in the workflow needs to have a different namespace name, to identify groups of pods to apply different completion criteria or measure them (like in our case). It's also possbile to use labels, but namespaces are the default and simpler option.

Typically, namespace are created implicitly within each operation, but since our time measurement spans over many operations and their names have to be provided, we need to name them explicitly.

Reply:
Acutally this is an answer to the question you asked when we had a chat. Why we don't measure time just for regular (`measure`) pods, I covered in other comment.

Reply:
Sorry, I think I missed the answer to this question. Can you re-post it here again?

Reply:
In this test case I measure the summarized time of both preemption and regular pods scheduling, as they will interleave with each other once async preemption is implemented (the async part will interleave). I assume here that the time will be measured at the beginning of the preemption process, which may require additional PR.

Not mesuring both would cause dividing the summarized scheduling time over subset of nodes, so lower the final throughput.